docs/4.2. Behaviour - Querying.md

```diff
@@ -10,25 +10,25 @@ Where the behaviour associated with an API attribute is not sufficiently clear f
+`secure` Indicates whether the WebSocket connection is provided via an encrypted connection (`ws://` vs. `wss://`). Unless otherwise indicated the value of this attribute SHOULD be `false` if the API is being presented via HTTP, and `true` for HTTPS. Query API clients MAY choose to specify the opposite value in requests for subscriptions, however they will receive a 400 response code unless the Query API explicitly supports a mismatch between encrypted HTTP and WebSocket connections.
 
 `authorization` Indicates whether the WebSocket connection requires authorization in order to connect. Use of authorization is likely to be a deployment decision and be the same for all subscriptions exposed from a single Query API, and for the RESTful API itself. Query API clients MAY specify a requested value for this attribute, but in most circumstances this will result in a 400 response code if the Query API is operating in the opposite mode.
 
 ## Creating a WebSocket subscription
 
 In order to connect to a WebSocket, a client MUST first request a subscription of a particular type and with particular query parameters by performing a POST as defined in the [Query API](../APIs/QueryAPI.raml) subscriptions documentation. Use of GET requests to find existing suitable WebSocket connections is strongly discouraged.
 
-There is no mandated URL base path for servers to use when creating WebSocket connections. Instead clients SHOULD observe the value of 'ws_href' which is returned by their subscription request in order to identify what to connect to.
+There is no mandated URL base path for servers to use when creating WebSocket connections. Instead clients SHOULD observe the value of `ws_href` which is returned by their subscription request in order to identify what to connect to.
 
 WebSockets created for specific clients will be cleaned up automatically if they disconnect.
 
 Connection upgrade procedures permitting an existing HTTP query to transition to a WebSocket are not currently supported, but could be in the future.
 
 ### Query API Behaviour
 
-Long-lived connections to the Query API are supported via WebSockets which can be set up via the /subscriptions resource. Query APIs MAY additionally support the HTTP 'Upgrade' header sent by clients to upgrade an HTTP GET request to a WebSocket. In cases where this is performed, a corresponding entry in /subscriptions MUST also be created with matching query parameters.
+Long-lived connections to the Query API are supported via WebSockets which can be set up via the `/subscriptions` resource. Query APIs MAY additionally support the HTTP `Upgrade` header sent by clients to upgrade an HTTP GET request to a WebSocket. In cases where this is performed, a corresponding entry in `/subscriptions` MUST also be created with matching query parameters.
```

Comment:
If we're backticking HTTP verbs...

```suggestion
Long-lived connections to the Query API are supported via WebSockets which can be set up via the `/subscriptions` resource. Query APIs MAY additionally support the HTTP `Upgrade` header sent by clients to upgrade an HTTP `GET` request to a WebSocket. In cases where this is performed, a corresponding entry in `/subscriptions` MUST also be created with matching query parameters.
```

Reply:
I decided to backtick verbs right at the end of yesterday's session, but don't feel strongly about them.